pennylane/devices/default_qubit_jax.py

```diff
@@ -196,6 +196,7 @@ def __init__(self, wires, *, shots=None, prng_key=None, analytic=None):
         del self._apply_ops["PauliY"]
         del self._apply_ops["Hadamard"]
         del self._apply_ops["CZ"]
+        self.operations = self.operations.copy()
```

Comment:
CC @Qottmann - to confirm, will this have any unexpected implications?
@timmysilv introduced a copy here because we were actually adding `ParametrizedEvolution` as a supported op to anything that inherts from `DefaultQubit`.

Reply:
What is that line intended for? I dont see any obvious implications and tests seem to pass as well

Reply:
It's relevant for the next line:
```python
self.operations.add("ParametrizedEvolution")
```

Reply:
Here's a summary of the problem. On master, you'd get:
```pycon
>>> qml.device("default.qubit.autograd", wires=1).supports_operation("ParametrizedEvolution")
False
>>> qml.device("default.qubit.jax", wires=1).supports_operation("ParametrizedEvolution")
True
>>> qml.device("default.qubit.autograd", wires=1).supports_operation("ParametrizedEvolution")
True
```

Creating a `default.qubit.jax` instance changes the entire DefaultQubit class until you restart your kernel. I have confirmed that on this branch, that last `True` is actually `False` (as it should be) because the jax device updates its copied operations list. I'm quite certain this is correct

Reply:
Aha! Nice catch! 🙂 I dont see any problem for pulse functionality 👍